test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -312,6 +312,21 @@ def setUp(self):
     echo "Devices directory should be populated"
 fi
 sleep 2
+"""
+        self.check_gpu_script = """#!/bin/bash
+#PBS -joe
+
+jobnum=${PBS_JOBID%%.*}
+devices_base=`grep cgroup /proc/mounts | grep devices | cut -d' ' -f2`
+if [ -d "$devices_base/propbs" ]; then
+    devices_job="$devices_base/propbs/$PBS_JOBID"
+else
+    devices_job="$devices_base/propbs.slice/propbs-${jobnum}.*.slice"
+fi
+device_list=`cat $devices_job/devices.list`
+grep "195" $devices_job/devices.list
+echo "There are `nvidia-smi -q -x | grep "GPU" | wc -l` GPUs"
+sleep 2
```

Comment:
Might want to extend this sleep a bit. Things happen rather slowly on certain systems.